src/components/header/Header.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components';
+
+const HeaderContainer = styled.div`
+  background-color: #e4e4ee;
+  display: flex;
+  justify-content: center;
+  padding: 10px;
+  h1 {
+    font-weight: lighter;
+    font-size: 25px;
+  }
+  .fa-bars {
+    position: absolute;
+    left: 2%;
+    cursor: pointer;
+  }
+  .fa-bars:hover {
+    color: #302f2f;
+  }
```

Comment:
These are a nice touch. To get a similar effect without introducing a new (outside the theme) color you could do `filter: opacity(0.7)`